src/test/java/com/rocana/lucene/codec/v1/TestBlockPostingsFormat.java

```diff
@@ -73,9 +71,9 @@ public void testFinalBlock() throws Exception {
 
     DirectoryReader r = DirectoryReader.open(w);
     assertEquals(1, r.leaves().size());
-    FieldReader field = (FieldReader) r.leaves().get(0).reader().fields().terms("field");
+    RocanaFieldReader field = (RocanaFieldReader) r.leaves().get(0).reader().fields().terms("field");
     // We should see exactly two blocks: one root block (prefix empty string) and one block for z* terms (prefix z):
-    Stats stats = field.getStats();
+    RocanaStats stats = field.getStats();
```

Comment:
Interestingly adding our postings format to the SPI file made these tests (originally from Lucene's code) use our forked RocanaFieldReader and RocanaStats instead of the Lucene originals, which is excellent.

I had to make this change or the test failed with a ClassCastException.